### PR TITLE
add EXMO exchange

### PR DESCRIPTION
**Pull request by pushistik**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "bitfinex-api-node": "^1.2.1",
     "co-fs": "^1.2.0",
     "commander": "^2.13.0",
+    "crypto-js": "^3.1.9-1",
```
Comment (askmike) [inline package.json]: can you add this dep to gekko broker instead? Which means inside this package instead: https://github.com/askmike/gekko/blob/develop/exchange/package.json

Else people won't be able to install gekko broker as a standalone package: https://www.npmjs.com/package/gekko-broker

**Review by askmike — CHANGES_REQUESTED** [review]
Please update the dep in the package.json!

**Comment by askmike** [comment]
Perfect, I will test this asap!

**Comment by askmike** [comment]
Apologies for the delay, I'm super. busy. will hopefully look at this later this week!

**Comment by pushistik** [comment]
Is any help required of me?

**Review by askmike — APPROVED** [review]

**Comment by askmike** [comment]
Apologies for the delayed merge.

**Pull request merged**